extensions/default/src/DicomWebDataSource/index.js

```diff
@@ -212,13 +210,14 @@ function createDicomWebApi(dicomWebConfig, userAuthenticationService) {
 
     store: {
       dicom: async (dataset, request) => {
-        wadoDicomWebClient.headers = getAuthrorizationHeader();
+        clientManagerObj.setWadoHeaders(2);
```

Comment:
It is not clear what the `2` argument means here. Consider simply having a ClientManager.setAuthorizationHeader method that is used here and maybe for line 105 (for Qido) and anywhere else the authorization header is needed.